src/app/shared/formly-address-forms/components/formly-address-form/formly-address-form.component.ts

```diff
@@ -20,7 +21,7 @@ export class FormlyAddressFormComponent implements OnInit, OnChanges {
 
   countryCode = '';
 
-  countries$: Observable<{ value: string; label: string }[]>;
+  countries$: Observable<SelectOption[]>;
```

Comment:
Good that there's a type, we should probably move it to shared/formly at some point